src/navigators/HomeStack.tsx

```diff
@@ -39,7 +38,6 @@ export function HomeStack() {
         name="Home/Circle/Discussion"
         component={CircleDiscussionScreen}
       />
-      <Stack.Screen name="Home/Circle/Thread" component={CircleThreadScreen} />
```

Comment:
Why is this being removed?

Reply:
I had a bunch of trouble with merge conflicts.  I believe that is an artifact.